src/FieldArray.tsx

```diff
@@ -9,8 +9,11 @@ import {
 } from './types';
 import { getIn, isEmptyChildren, isFunction, setIn } from './utils';
 
-export type FieldArrayRenderProps = ArrayHelpers & {
-  form: FormikProps<any>;
+export type FieldArrayRenderProps<
+  FormValue = any,
+  FieldArrayValue = any
```

Comment:
Note that I'm just a lurker here, not a Formik developer.

But it seems the convention is that type variables are either called `T` or short single-letter words.

So for consistency, maybe `FormValue` -> `Values` and `FieldArrayValue` -> `T`

Reply:
Correct.